tests/openssl/CMakeLists.txt

```diff
@@ -0,0 +1,494 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(OPENSSL_DIR ${PROJECT_SOURCE_DIR}/3rdparty/openssl/openssl)
+
+add_subdirectory(host)
+if (BUILD_ENCLAVES)
+  add_subdirectory(enc)
+endif ()
+
+macro (copy_test_data OLD_FILE NEW_FILE)
+  if (UNIX)
+    add_custom_command(OUTPUT ${NEW_FILE} COMMAND ${CMAKE_COMMAND} -E copy
+                                                  ${OLD_FILE} ${NEW_FILE})
+  else ()
+    # On Windows, invoking the dos2unix utility from git to replace
+    # CRLF by LF in the test file.
+    add_custom_command(OUTPUT ${NEW_FILE} COMMAND ${OE_DOS2UNIX} -n ${OLD_FILE}
+                                                  ${NEW_FILE})
+  endif ()
+
+  # Add the copied files to the list so that the custom target
+  # openssl_test_data can take dependency on.
+  list(APPEND test_data_list ${NEW_FILE})
+endmacro (copy_test_data)
+
+# Skip on Windows if dos2unix is not supported.
+if (UNIX OR OE_DOS2UNIX)
+  # Copy the following test data from the OpenSSL repository so
+  # that we can enforce LF end-of-lines on Windows.
+  copy_test_data(${OPENSSL_DIR}/test/recipes/30-test_evp_data/evpcase.txt
```

Comment:
I've added the comments on why only copying these files.

Reply:
I share the same concerns about maintainability. 
Instead of selectively fixing up these files, can the line ending differences be addressed via
- Some git option for submodules
- Or via a script that fixes line endings for all files by iterating through them

It is probably fine as-is for this PR since you have already done the work to figure out the files.

These specific list of tests is an example of the complexity that it would be better to not add on to the OpenEncalve repository.

Reply:
I couldn't find a good way to enforce that on submodule.  I'd like to have other issue to investigate that.